src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -74,6 +74,7 @@
         ("swin", "ViTFeatureExtractor"),
         ("swinv2", "ViTFeatureExtractor"),
         ("table-transformer", "DetrFeatureExtractor"),
+        ("timesformer", "VideoMAEFeatureExtractor"),
```

Comment:
> Thanks for your work, looks great to me. Pinging @amyeroberts for the feature extractor part; I guess we can't replace it with `VideoProcessor` for the moment?

@NielsRogge @fcakyon Looking quickly over the PR, all I think needs to happen is to add the equivalent line to this in `image_processing_auto.py` 
```
("timesformer", "VideoMAEImageProcessor"),
```

I can't find any specific Qs about the feature processor - but might have missed them. What is the `VideoProcessor` in reference to?

Reply:
Well since this is a video model, I'm not sure the term `ImageProcessor` makes sense, `VideoProcessor` might make more sense.

Reply:
Videos are made of images so it's not that far-fetched. I'd avoid building too many base objects if possible.

Reply:
@amyeroberts I have made the required updates 👍